src/model/entities/timeline.ts

```diff
@@ -34,6 +34,11 @@ export enum TimelineObjectType {
   GROUP = 'group',
 }
 
+export interface LookAheadTimelineObject extends TimelineObject {
+  isLookahead: boolean
```

Comment:
If you have an object with this type, when should/would this be `false`?

Reply:
Also should it be `Lookahead` or `LookAhead`, as it is in the interface name?

Reply:
In theory, it should never be false when dealing with this type.
`isLookahead` and `lookaheadLayer` are both used by TSR when resolving the Timeline and building commands for the devices.
So unfortunately we can't change the names. I will add a comment to make this dependency less hidden.